mssql-windows/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# escape=`
+
+# adapted from https://github.com/microsoft/mssql-docker/blob/e4afa866646c25decef32f8946a995074c3251e7/windows/mssql-server-windows-express/dockerfile
+
+ARG WINDOWS_SERVER_VERSION=ltsc2019
+FROM mcr.microsoft.com/windows/servercore:${WINDOWS_SERVER_VERSION}
```

Comment:
Why re-create their SQL image instead of building on top of it?

Reply:
Their SQL image is built against Windows Server 2016, which cannot be run natively on Windows Server 2019.

Reply:
So Windows containers are all built in an OS version dependant way?  🤦‍♂ 

Thanks for the explanation.

Reply:
Yes, the native windows container format requires the container image to have the same base OS as the host. This isn't true for HyperV, but I'm trying to avoid that requirement.